src/smc-webapp/projects.cjsx

```diff
@@ -525,7 +525,8 @@ class ProjectsStore extends Store
             return 0
         others = []
         for i in v
-            if not i.deleted and (show_hidden or not i.get('users').get(account_id).get('hide'))
+            # Deleted projects have a map node " 'deleted': true ". Standard projects do not have this property.
+            if (not i.get('deleted')) and (show_hidden or not i.get('users').get(account_id).get('hide'))
```

Comment:
So to summarize, the net relevant change is to replace `i.deleted` by `i.get(‘deleted’)`, which was needed because i is a list of immutable.js objects?